policyengine_us/parameters/gov/hhs/medicaid/eligibility/categories/infant/income_limit.yaml

```diff
@@ -65,6 +67,7 @@ MT:
   2021-01-01: 1.48
 NC:
   2021-01-01: 2.15
+  2024-01-01: 2.16
```

Comment:
Running through the reference, it might be more accurate to set the start date to
2023-12-01:
![image](https://github.com/user-attachments/assets/01f5f628-70b0-46be-8064-fd1b0e299832)